ignite/distributed/comp_models/native.py

```diff
@@ -354,8 +354,8 @@ def spawn(  # type: ignore[override]
             nproc_per_node: int = 1,
             nnodes: int = 1,
             node_rank: int = 0,
-            master_addr: Optional[str] = None,
-            master_port: Optional[int] = None,
+            master_addr: Optional[str] = "127.0.0.1",
+            master_port: Optional[int] = 2222,
```

Comment:
This fixes the issue, but if user defines `init_method` as something unrelated to tcp store (e.g. filestore), then seeing defined `master_addr` and `master_port` and `init_method` together can make things a bit cumbersome...
How about keeping them as None (same as previously) but defining as you do in case of init_method is None and they are undefined (are Nones) ?

Reply:
Yes I agree I was also thinking about that, but didn't know clearly the impact for the other methods, will fix this , thanks for your comment @vfdev-5